package.json

```diff
@@ -4,7 +4,8 @@
   "description": "Back-End Repo for Team 1 of Baboon/Bald Eagle Practicum",
   "main": "app.js",
   "scripts": {
-    "dev": "nodemon src/server.js"
+    "dev": "nodemon src/server.js",
+    "start": "node src/server.js"
```

Comment:
I would recommend using nodemon when running npm start. That way whenever server files are changed the server will reload.